_pages/docs/tutorials/beginner-tutorial.mdx

```diff
@@ -0,0 +1,213 @@
+
+## What is an API?
+
+API stands for Application Programming Interface. The key to an APi is that it allows applications to talk to each other. Here are some current examples of API's used today.
+* Uber drivers have Google Maps built into their Uber app for naviation.
+* Duolingo allows users to log in using their Facebook information.
+* Many applications use Stripe for processing payments. 
+
+API's are powerful. Let's take the Uber example above. It would take a lot of time and engineers to develop a map feature like similar Google Maps within the Uber app. Uber uses Google Maps' API to access it within their own app. Of course, Uber pays a portion of their sales to Google in return for using their API. 
+
+## Let's Get Started
+
+For you to communicate with KittyCAD's website, you must have an API key. When you make an API command, you will pass this APi key along with your command, so that we can verify it's you. 
+
+If you have already generated an API key, you will see it shown below. If you see "No API keys have been generated yet". In order for you to see your API key, you must do the following:
+* Create an account. If you haven't already, you can [create an account here](/account)
+* Sign in. Once you've signed up for an account, make sure you are signed in. 
+* Generate an API key. You can either go into your [account settings](/account?tab=api_tokens) to generate an API key, or you can press the button below and we will take care of it for you!
+
+Generate API Key button here
+
+## The Litterbox
+
+Below is what we call our "Litterbox" (other sites tend to refer to it as a "Sandbox"). Essentially, this is an area in which you can try out code without any additional steps. Below is an example of a Python script that uses KittyCAD's API to convert an OBJ file to an STL file.
+
+{/* __PRE-FETCH__: tutorials/getting_started/getting_started */}
+
+<InlineLitterbox path="tutorials/getting_started/getting_started" />
+
+Now try modifying the code and executing it again. As an example, delete the "#" to the left of `"print(<your-name>, congrats! Your STL conversion was successful):"` and replace `<your-name>` with your first name.
+
+<TabbedEditor
+    tabs={[
+        {
+            title: '',
+            range: [0, 0],
+            lang: 'py',
+            code: `# Try adding your name by changing the text below to \n# print(<your-name>, congrats! Your STL conversion was successful):`,
+        },
+    ]}
+/>
+
+Then execute. If you came to this tutorial from [Account Onboarding Progress](/account?tab=onboarding_progress) than you'll need to modify and execute the code in order for it to count as completing the second task.
+
+## Executing From Your Computer
+
+* First, we want to make sure you have the right tools to make your first API command from your computer. We will provide you with most everything you need. There are a few files that you will need to download from the KittyCAD repository:
+Download Here
```

Comment:
We should make sure this link is there before this goes live. I did merge the [draft logic](https://github.com/KittyCAD/website/pull/886) already, so could you either add this link, or rename this file to `draft-beginner-tutorial.mdx`